ci/Jenkinsfile

```diff
@@ -112,20 +112,15 @@ timeout(90) {
 
         stage('Upload apk for e2e tests') {
           if (env.CHANGE_ID != null) {
-
-              withCredentials([
-                  string(credentialsId: 'SAUCE_ACCESS_KEY', variable: 'key'),
-                  string(credentialsId: 'SAUCE_USERNAME', variable: 'username')
-              ]) {
-                def apk_name = env.CHANGE_ID + '.apk'
-                sh('curl -u ' + username+ ':' + key + ' -X POST -H "Content-Type: application/octet-stream" https://saucelabs.com/rest/v1/storage/' + username + '/' + apk_name + '?overwrite=true --data-binary @android/app/build/outputs/apk/release/app-release.apk')
-              }
-
+              env.SAUCE_LABS_APK = 'im.status.ethereum-e2e-' + env.CHANGE_ID + '.apk'
               withCredentials([
+                  string(credentialsId: 'SAUCE_ACCESS_KEY', variable: 'SAUCE_ACCESS_KEY'),
+                  string(credentialsId: 'SAUCE_USERNAME', variable: 'SAUCE_USERNAME'),
```

Comment:
Wouldn't it make more sense to do a separate `withCredentials` for the `fastlane android saucelabs` so it's clear which credentials are for which call?

Reply:
I tried it first, but then it is harder to read :(